lib/mix/tasks/openapi.spec.yaml.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Mix.Tasks.Openapi.Spec.Yaml do
+  @moduledoc """
+  Serialize the given OpenApi spec module to a YAML file.
+
+  ## Examples
+
+      $ mix openapi.spec.yaml --spec PhoenixAppWeb.ApiSpec apispec.yaml
+      $ mix openapi.spec.yaml --spec PhoenixAppWeb.ApiSpec --vendor-extensions=false
+
+  ## Command line options
+
+  * `--spec` - The ApiSpec module from which to generate the OpenAPI YAML file
+
+  * `--vendor-extensions` - Whether to include open_api_spex OpenAPI vendor extensions
+    (defaults to true)
+
+  """
+  use Mix.Task
+  require Mix.Generator
+
+  @default_filename "openapi.yaml"
+
+  @impl Mix.Task
+  def run(argv) do
+    Mix.Task.run("app.start")
+    OpenApiSpex.ExportSpec.call(argv, &encode/2, @default_filename)
+  end
+
+  defp encode(spec, opts) do
+    spec
+    |> OpenApiSpex.OpenApi.yaml_encoder().encode(opts)
```

Comment:
The `yaml_encoder` can return `nil` if the `ymlr` dependency is not added. My suggestion is to check whether it's `nil` and if so `Mix.raise` an informative error message asking the user to ensure they've added the dependency.

Reply:
Fixed